app/posts/check-the-record-of-a-teacher/2023-09-01-inviting-people-to-use-the-service.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Inviting people to use the service
+date: 2023-09-01
+---
+
+Initially we’ll only allow people from teacher recruitment agencies to use the Check the record of a teacher service.
+
+The current Employer Access service has around 1000 users from around 350 recruitment agencies.
+
+We’ve chosen this group because:
+
+- they do not need the ‘add to my organisation’ function in Employer Access - we have not yet decided whether to offer a similar function in the new service
+- they only need to search for teachers in the Children’s Barred List - if they needed to search for other people, they’d need the separate service which will not launch immediately
```

Comment:
Can we elaborate on this? What is covered in the Children’s Barred List, what is the "separate service" and what does that cover?